### PR TITLE
Bookstore: Styling

### PR DESCRIPTION
**Pull request by anyars-encarta**
In this project:

- [ ] I added missing JSX elements to my project.
- [ ] The final design matches the Zeplin's design (this includes all elements + the same styling)
- [ ] I used the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/styles/Book.css [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    transform: rotate(180deg);
+  }
+
+  100% {
+    z-index: 100;
+    transform: rotate(315deg);
+  }
+}
+
+.progress-percent {
```
Comment (tufoinnkuo10) [inline src/styles/Book.css:121]: - [ ] wonderful design you got there. let's make the final touches, so they align with the zeplin design. Please kindly align the main page book heading and the user logo placeholder with the book cards like in the original design.
yours
Screenshot_20
original
Screenshot_21

File: src/styles/Book.css [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+  width: 4.25rem;
```
Comment (tufoinnkuo10) [inline src/styles/Book.css:122]: - [ ] kindly shift the add book button a little to the right to it aligns with the book cards as well like in the the design. nice work.
yours
Screenshot_19
original
Screenshot_18

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @anyars-encarta 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Design well followed

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/styles/Book.css (hunk 1 above)
Comment (anyars-encarta) [reply src/styles/Book.css:121]: Wonderful and critical observation. Thanks for finding that out. I will correct that as you suggested.

File: src/styles/Book.css (hunk 2 above)
Comment (anyars-encarta) [reply src/styles/Book.css:122]: I must say that I really admire your attention to detail. Correcting that asap!

File: src/components/Navbar.js [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
+  <div className="nav-master">
+    <div className="nav-container">
+      <nav className="nav-group">
+        <h1 className="header-title">Bookstore CMS</h1>
+        <ul>
+          <li><NavLink className="books" to="/">BOOKS</NavLink></li>
+          <li><NavLink className="categories" to="/categories">CATEGORIES</NavLink></li>
+        </ul>
+      </nav>
+      <div className="user" />
```
Comment (shahadat3669) [inline src/components/Navbar.js:15]: - [ ] [Optional] Well! Your icon is a nice addition, but there are a few adjustments needed to match the design more closely. Currently, you've added a shadow to the icon, which is not present in the design. I recommend removing the shadow. Moreover, kindly ensure that the icon's background color matches the design specifications.

| Your | Design |
| --- | --- |
|  image| image |

By making these changes, your icon will align better with the intended design, creating a more cohesive and visually appealing user interface. Keep up the good work! 👍🎨✨

File: src/styles/Book.css [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.update-progress {
```
Comment (shahadat3669) [inline src/styles/Book.css:278]: - [ ] [Optional] Well! Paying close attention to design details is crucial for achieving a polished look. Currently, the `.update-progress` button has a font-weight of 400, but according to the design, it should be 300. To ensure your project aligns with the design as closely as possible, please update your code to set the font weight to 300 for this button.

By making this adjustment, you'll enhance the consistency of your project's appearance with the design guidelines. Keep up the great work! 👍🎨💼

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
.

**Review by shahadat3669 — APPROVED** [review]
Hi @anyars-encarta , 

Your project is complete!  There is nothing else to say other than it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ No ID selectors for multiple elements
✅ No !important overusing
✅ No inline styling

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table  width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**